src/app/(unauthenticated)/invest/[slug]/page.tsx

```diff
@@ -7,7 +17,19 @@ type Props = {
 export default async function InvestPage({ params }: Props) {
   return (
     <BaseLayout>
-      <div>Invest details {params.slug}</div>
+      <div className='flex flex-col justify-center items-center max-w-screen-xl w-full'>
+        <Header title={ProjectHeaderData.title} subtitle={ProjectHeaderData.subtitle} />
+        <div className='flex flex-row w-full h-full gap-6'>
```

Comment:
I think we can get rid of setting width to 72% and 28% in another place ( I saw it somewhere).
We can apply flex, set the gap and then fill out the space to be similar to Figma.
(setting max-width on component and take full width for image or Project Box on the right).

And regarding responsiveness just  change it from `flex flex-row w-full h-full gap-6` to `flex flex-col md:flex-row w-full h-full gap-6 md:gap-10`